_issues/rapist.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title: "Didn't she get a rapist acquitted, and then laugh about the girl who was raped?"
+categories: controversy
+---
+
+**Q:** Didn't Hillary get a rapist acquitted, and then laugh about the girl who was raped?  
+
+**A**: In the second Presidential debate, Donald Trump claimed "Her client she represented
+got him off and she is seen laughing on two occasions laughing at the girl who was raped." This is [completely false](http://www.politico.com/blogs/2016-presidential-debate-fact-check/2016/10/trump-is-wrong-hillary-clinton-did-not-laugh-about-the-rape-of-a-12-year-old-229455#ixzz4MiTn0Cwt). Here are the facts:
+
+* In 1975, Hillary was **appointed by a judge** to defend a homeless man who was being tried for the rape of a 12 year old girl. 
```

Comment:
This may be nit-picky but I feel like people might be more familiar with something like "...Hillary was the court appointed attorney to a homeless man ..." It's a small thing, but "court appointed attorney is how its referred to on television, so I feel like it might be more recognizable that way.